create_sample_data.py

```diff
@@ -73,9 +74,19 @@ def check_dataset_lock(dataset_dbid):
                 ## This lock check and sleep is here to prevent the dataset from being permanently
                 ## locked because a tabular file was uploaded first.
                 check_dataset_lock(dataset_dbid)
-                file_metadata = {}
                 # TODO: Think more about where the description comes from. A "sidecar" file as proposed at https://github.com/IQSS/dataverse/issues/5924#issuecomment-499605672 ?
-                #file_metadata['description'] = 'Sidecar?'
+                # L.A.: I implemented something along these lines - an (optional) directory called ".filemetadata" 
+                # in the dataset directory, where files containing extra json filemetadata records may be 
+                # placed for each of the files in the "files" directory. 
+                # (since the file names must be unique per dataset, even with folders, the .filemetadata 
+                # directory structure is flat)
```

Comment:
Do file names really need to be unique? I could swear I created a "file.txt" over and over in multiple directories it the same dataset.

Reply:
You are correct, files with the same name can occur in different folders (i.e., only the combination of "folder name" + "file name" must be unique). 
I checked in a change that accommodates this hierarchy.

Reply:
@landreev great! It sounds like the only thing remaining is to remove the "file names must be unique per dataset" comment above. But what should we do about the "This file already exists in the dataset" note I left in the README? Is it still valid? Should it be removed too?

Reply:
ok, this was done.